docs/install-cpu-pip.rst

```diff
@@ -1,10 +1,10 @@
 .. code-block:: bash
 
-    pip3 install -U pip
-    pip3 install -U setuptools wheel
+    pip install -U pip
+    pip install -U setuptools wheel
 
     # CPU version of pytorch has smaller footprint - see installation instructions in
     # pytorch documentation - https://pytorch.org/get-started/locally/
-    pip3 install torch==1.10.1+cpu -f https://download.pytorch.org/whl/cpu/torch_stable.html
+    pip install "torch>=1.0,<1.11+cpu" -f https://download.pytorch.org/whl/cpu/torch_stable.html
```

Comment:
`1.0` left boundary looks too wide: I would suggest doing at least `>=1.10`, otherwise we have a risk having environments with `1.0.8`, which are too old and possibly incompatible with other libraries in the bundle (`fastai` for instance)

Reply:
I am merely matching our version range we have in setup.py. Changing the range can be a separate conversation for v0.5